### ISSUE TITLE
New installations of this addon use classnames 2.3.0 which breaks apps in production

### ISSUE THREAD

**Issue by Aierie** [PrecisionNutrition/ember-class-names-helper#6]
### Issue
As of yesterday, when this addon is installed and used with the latest versions of ember, websites crash in production with the error: 

```
Uncaught TypeError: arg.toString is not a function
    at classNames (index.js:39)
    at classNames (class-names.js:10)
    at index.js:3961
    at deprecateMutationsInTrackingTransaction (validator.js:180)
    at index.js:3960
    at SimpleClassicHelperManager.getValue (index.js:3971)
    at manager.js:949
    at reference.js:161
    at track (validator.js:820)
    at valueForRef (reference.js:160)
```

The addon resolves `classnames` to version 2.3.0 which tries to call the `toString` method on the `hash` argument, which doesn't exist.

https://github.com/JedWatson/classnames/blob/f2379c3d1eae8c9d6527279bc6a0630a2e78b051/index.js#L39

### Quick reproduction
basically just `ember new` and `ember install ember-class-names-helper` + simple changes to the application template:

https://github.com/Aierie/classnames-error-reproduction

**Comment by jherdman** [comment]
Released in 0.2.1.
